Lib/uuid.py

```diff
@@ -403,9 +403,16 @@ def _get_command_stdout(command, *args):
 #
 # See https://en.wikipedia.org/wiki/MAC_address#Universal_vs._local
 
+
 def _is_universal(mac):
     return not (mac & (1 << 41))
 
+# MAC adresses that are blacklisted for finding a node id
+_MAC_BLACKLIST={
+    # (GH-85724) Hardcoded MAC adres for all Intel MacBook's with Touch Bar.
+    int("ac:de:48:00:11:22".replace(":", ""), 16),
```

Comment:
Since there is a single MAC address, I don't think that using a set is useful here.

I suggest ``_MACBOOK_TOUCHBAR_MAC = ...``.

If you prefer a set, please rename it to "BLOCKLIST", rather than "BLACKLIST" ;-)